app/lib/actions/katello/repository/remove_content.rb

```diff
@@ -46,8 +49,9 @@ def humanized_name
         end
 
         def run
-          output[:task_id] = ForemanTasks.async_task(Actions::Katello::Repository::MetadataGenerate,
-                                                     ::Katello::Repository.find(input['repository_id'])).id
+          output[:task_id] = ForemanTasks
+            .sync_task(Actions::Katello::Repository::MetadataGenerate,
+                       ::Katello::Repository.find(input[:repository_id])).id
```

Comment:
rubocop is set to 200:  https://github.com/Katello/katello/blob/master/.rubocop_todo.yml#L44

I don't know why hound is complaining, did re-pushing not help?